### PR TITLE
Distributed Workloads v0.1.0 release

### PR DESCRIPTION
**Pull request by anishasthana**
cc @tedhtchang @jbusche @sutaakar 

I will create a tag + release after this is merged.

### CODE REVIEW

**Review by tedhtchang — CHANGES_REQUESTED** [review]
Since the codeflare-sdk has removed the min/max_worker in the clusterConfig, we may need to replace min_worker and max_worker with workers=2
[here](https://github.com/anishasthana/distributed-workloads/blob/v0.1.0_release/tests/resources/mnist_ray_mini.ipynb) and also the [quicker start doc](https://github.com/anishasthana/distributed-workloads/blob/v0.1.0_release/Quick-Start.md#run-the-demo-notebook) as well.

## Test steps
Install latest ODH operator
Install and verify codeflare-operator is v0.1.0
verified mcad image quay.io/project-codeflare/mcad-controller:release-v1.33.0
verified instascale image quay.io/project-codeflare/instascale-controller:v0.0.6
verified codeflare-sdk 0.6.1 (I don't think it is necessary to install locally)
Ran the [e2e test](https://github.com/anishasthana/distributed-workloads/blob/v0.1.0_release/tests/README.md)
✘ FAILURE after 1203.000s: operator-tests/distributed-workloads/tests/basictests/distributed-workloads.sh:125: executing 'oc get appwrapper mnisttest -n opendatahub -ojsonpath='{.status.state}'' expecting any result and text 'Running'; re-trying every 10s until completion or 1200.000s: the command timed out
```
papermill.exceptions.PapermillExecutionError:
---------------------------------------------------------------------------
Exception encountered at "In [2]":
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In [2], line 2
      1 # Create our cluster and submit appwrapper
----> 2 cluster = Cluster(ClusterConfiguration(namespace='opendatahub', name='mnisttest', min_worker=2, max_worker=2, min_cpus=2, max_cpus=2, min_memory=4, max_memory=4, gpu=0, instascale=False))

TypeError: __init__() got an unexpected keyword argument 'min_worker'
```

**Comment by Maxusmusti** [comment]
Yeah specifically, `min_workers` and `max_workers` have been replaced with `num_workers`, and `gpu` is now also called `num_gpus`

File: tests/resources/mnist_ray_mini.ipynb [hunk 1]
```diff
@@ -25,7 +25,7 @@
    "outputs": [],
    "source": [
     "# Create our cluster and submit appwrapper\n",
-    "cluster = Cluster(ClusterConfiguration(namespace='opendatahub', name='mnisttest', min_worker=2, max_worker=2, min_cpus=2, max_cpus=2, min_memory=4, max_memory=4, gpu=0, instascale=False))"
+    "cluster = Cluster(ClusterConfiguration(namespace='opendatahub', name='mnisttest', num_workers=2, min_cpus=2, max_cpus=2, min_memory=4, max_memory=4, gpu=0, instascale=False))"
```
Comment (Maxusmusti) [inline tests/resources/mnist_ray_mini.ipynb:28]: change `gpu` to `num_gpus`

File: Quick-Start.md [hunk 2]
```diff
@@ -136,13 +136,12 @@ cluster_config = ClusterConfiguration(
...
     machine_types = ["m4.xlarge", "g4dn.xlarge"]
-    min_worker=2, 
-    max_worker=2, 
+    num_workers=2,
     min_cpus=2, 
     max_cpus=2, 
     min_memory=8, 
     max_memory=8, 
-    gpu=1, 
+    num_gpu=1,
```
Comment (Maxusmusti) [inline Quick-Start.md:144]: change `num_gpu` to `num_gpus`

**Review by Maxusmusti — CHANGES_REQUESTED** [review]

**Review by Maxusmusti — APPROVED** [review]

**Review by tedhtchang — APPROVED** [review]

**Comment by tedhtchang** [comment]
@anishasthana I was able to pass the e2e test on the same OCP cluster. Looks good. Thanks.

**Pull request merged**